packages/js/src/ui/components/primitives/Tabs/TabsTab.tsx

```diff
@@ -0,0 +1,43 @@
+import { JSX, ParentProps, Ref } from 'solid-js';
```

Comment:
Maybe just `Tab.tsx`? The stuttering feels a bit weird.

Reply:
I'll follow Radix pattern: 
```
<Tabs.Root>
    <Tabs.List>
      <Tabs.Trigger />
    </Tabs.List>
    <Tabs.Content />
  </Tabs.Root>
```